src/Pages/category/Component/CategoryList/CategoryListItem/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+function CategoryListItem({
+  categoryList,
+  onChangeClickId,
+  handleOnchangeMenuFocus,
+  chgColorMenu,
+}) {
+  return (
+    <ListItemWrap>
+      {categoryList.map((list, index) => {
+        return (
+          <ListItem
+            onFocus={handleOnchangeMenuFocus}
+            onBlur={handleOnchangeMenuFocus}
+            key={index}
+          >
+            <Link
+              to=""
+              onClick={() => onChangeClickId(index)}
+              chgColorMenu={chgColorMenu}
+            >
+              {list.name}
+            </Link>
+          </ListItem>
+        );
+      })}
+    </ListItemWrap>
+  );
+}
+
+const ListItemWrap = styled.ul`
+  color: #424242;
+`;
+
+const ListItem = styled.li`
+  a:link {
+    color: red;
+    /* ${props => (props.chgColorMenu ? 'black' : 'red')} */
+  }
+  font-size: 14px;
+  & + & {
```

Comment:
```suggestion
  }
  
  font-size: 14px;
  
  & + & {
```
새로운 셀렉터 끼리는 한줄씩 띄워주세요